packages/job-worker/src/playout/quickLoopMarkers.ts

```diff
@@ -17,9 +20,74 @@ export async function handleSetQuickLoopMarker(context: JobContext, data: SetQui
 		async (playoutModel) => {
 			const playlist = playoutModel.playlist
 			if (!playlist.activationId) throw new Error(`Playlist has no activationId!`)
-			const wasQuickLoopRunning = playoutModel.playlist.quickLoop?.running
+			const oldProps = playoutModel.playlist.quickLoop
```

Comment:
It might be good for this to explicitly clone the props, as there is no guarantee that the implementation of `setQuickLoopMarker` will do so